packages/react/popover/src/Popover.tsx

```diff
@@ -255,21 +257,30 @@ const PopoverContentNonModal = React.forwardRef((props, forwardedRef) => {
         onEscapeKeyDown={composeEventHandlers(props.onEscapeKeyDown, () => {
           isPointerDownOutsideRef.current = false;
         })}
-        onPointerDownOutside={composeEventHandlers(props.onPointerDownOutside, (event) => {
-          const { originalEvent } = event.detail;
-          const isLeftClick = originalEvent.button === 0 && originalEvent.ctrlKey === false;
-          isPointerDownOutsideRef.current = isLeftClick;
-        })}
-        onInteractOutside={composeEventHandlers(props.onInteractOutside, (event) => {
-          // Prevent dismissing when clicking the trigger.
-          // As the trigger is already setup to close, without doing so would
-          // cause it to close and immediately open.
-          //
-          // We use `onInteractOutside` as some browsers also
-          // focus on pointer down, creating the same issue.
-          const targetIsTrigger = context.triggerRef.current?.contains(event.target as HTMLElement);
-          if (targetIsTrigger) event.preventDefault();
-        })}
+        onPointerDownOutside={composeEventHandlers(
+          props.onPointerDownOutside,
+          (event) => {
+            const { originalEvent } = event.detail;
+            const isLeftClick = originalEvent.button === 0 && originalEvent.ctrlKey === false;
+            isPointerDownOutsideRef.current = isLeftClick;
+          },
+          { checkForDefaultPrevented: false }
+        )}
+        onInteractOutside={composeEventHandlers(
```

Comment:
This differs in that it covers the focus where before we didn't need to, not sure if that makes much difference where this is concerned.